cfg.y

```diff
@@ -661,6 +681,9 @@ socket_def_param: ANYCAST { IFOR();
 				| REUSE_PORT { IFOR();
 					p_tmp.flags |= SI_REUSEPORT;
 					}
+				| WILDCARD_MATCH { IFOR();
```

Comment:
This is so with sockets we can do the same thing but we don't add the alias to the socket because then set_advertised_address does not work